plugins/feeds/public/hostsfile_emd.py

```diff
@@ -0,0 +1,52 @@
+from datetime import datetime, timedelta
+import logging
+
+from core.observables import Hostname
+from core.feed import Feed
+from core.errors import ObservableValidationError
+
+class HostsFile_EMD(Feed):
+	default_values = {
+		'frequency': timedelta(hours=4),
+		'source': 'https://hosts-file.net/emd.txt',
+		'name': 'Hosts_File_EMD',
```

Comment:
the `name` in the dict must be the same as the class name (otherwise this won't work with the scheduler; I agree some work should be done to avoid having to specify a constrained name). Also, to keep naming consistent, I would prefer to rename this to `HostsFileEMD` and drop the underscores.